README.md

```diff
@@ -1,25 +1,36 @@
 # CombineFirebase
 
+[![SwiftPM compatible](https://img.shields.io/badge/SwiftPM-compatible-informational)](#swift-package-manager)
 [![Version](https://img.shields.io/cocoapods/v/CombineFirebase.svg?style=flat)](https://cocoapods.org/pods/CombineFirebase)
 [![License](https://img.shields.io/cocoapods/l/CombineFirebase.svg?style=flat)](https://cocoapods.org/pods/CombineFirebase)
 [![Platform](https://img.shields.io/cocoapods/p/CombineFirebase.svg?style=flat)](https://cocoapods.org/pods/CombineFirebase)
 
+Handling Firebase asynchronous callbacks with Combine framework.
+
 ## Example
 
-To run the example project, clone the repo, and run `pod install` from the Example directory first.
+To run the example project, clone the repo, and run `pod install` from the Example directory first. See [Usage](#Usage) below for more information.
 
 ## Requirements
 
-Xcode 11.3
-
-Swift 5.1
+* Xcode 11.3
+* Swift 5.1
+* iOS/tvOS 13.0+, macOS 10.15+
 
 ## Installation
 
-CombineFirebase is available through [CocoaPods](https://cocoapods.org). To install
+#### Swift Package Manager
+`CombineFirebase` is available through [Swift Package Manager](https://swift.org/package-manager/). To install it, simply add `CombineFirebase` to an existing Xcode project as a package dependency:
+
+1. From the **File** menu, select **Swift Packages > Add Package Dependency...**
+2. Enter https://github.com/rever-ai/CombineFirebase into the package repository URL text field.
+3. Xcode should choose updates package up to the next version option by default.
+
+#### CocoaPods
+`CombineFirebase` is available through [CocoaPods](https://cocoapods.org). To install
 it, simply add the following line to your Podfile:
```

Comment:
I think it is reasonable to add beta warning and make cocoapods method as recommended for now.

Reply:
I just updated `README.md` in my last commit (e191eea), and also other minor stuffs.

Reply:
I agree. I just updated `README.md` in my last commit (e191eea) and also other minor stuffs.